railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -20,7 +20,9 @@ Rails.application.configure do
 
   <%- unless options.skip_sprockets? -%>
   # Compress JavaScripts and CSS.
+  <%- unless options.skip_javascript? -%>
```

Comment:
This is really nit-picky but what do you think about:

~~~erb
<%- unless options.skip_sprockets? -%>
<%- unless options.skip_javascript? -%>
# Compress JavaScripts and CSS.
config.assets.js_compressor = :uglifier
<%- else -%>
# Compress CSS.
<%- end -%>
~~~